__tests__/app.test.js

```diff
@@ -356,4 +356,32 @@ describe('/api/reviews/:review_id/comments', () => {
                 });
         });
     });
+});
+
+describe("/api/users", () => {
+    describe("GET", () => {
+        test("should return a table of user objects which have the correct properties", () => {
+            return request(app)
+                .get("/api/users")
+                .expect(200)
+                .then(({ body }) => {
```

Comment:
We need to think about checking the length of the array here before doing the forEach as an empty array would pass this test which clearly is not good. Before the forEach assert at the bare minimum the length is greater than 0. Better yet assert it to be the specific expected length of the array. 👍